microsoft-edge/dualengine/reference/dualenginesessionfactory.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Factory object for DualEngine interfaces.
+title: DualEngine Win32 C++ DualEngineSessionFactory
+author: MSEdgeTeam
+ms.author: edgededev
+ms.prod: microsoft-edge
+ms.date: 11/16/2023
+keywords: dual engine, dualengine, iemode, win32 apps, win32, edge, ie mode, edge html, DualEngineSessionFactory
+topic_type: 
+- APIRef
+api_name:
+- DualEngineSessionFactory
+- DualEngineSessionFactory.GetSpecifiedVersion
+- DualEngineSessionFactory.GetVersionedBrowserSession
+- DualEngineSessionFactory.TryUnlockFeature
+api_type:
+- COM
+api_location:
+- ieframe.dll
+---
+
+# interface DualEngineSessionFactory
+
+> [!IMPORTANT]
+> The DualEngine API is a limited access feature. Contact dualengineapiaccess@microsoft.com for more information.
+
+Factory object for DualEngine interfaces.
+
+## Summary
+
+ Members                        | Descriptions
+--------------------------------|---------------------------------------------
+[GetSpecifiedVersion](#getspecifiedversion) | Returns the known DualEngine interface version, if there is one.
+[GetVersionedBrowserSession](#getversionedbrowsersession) | Gets a DualEngine Session object of the requested version.
+[TryUnlockFeature](#tryunlockfeature) | Called to unlock the DualEngine interface.
+
+## Members
+
+#### GetSpecifiedVersion
+
+Returns the known DualEngine interface version, if there is one.
+
+> public HRESULT [GetSpecifiedVersion](#getspecifiedversion)(__out UINT * version) const
+
+#### Parameters
+* `version` A pointer to a UINT where the version will be stored if there is one. 
+
+#### Returns

```

Comment:
```suggestion
###### Returns
```
global

Reply:
today in .md, demoted this Returns heading and this Params heading from h4 to h6, like already for the other functions in this .md file.  Probably apply same change to source file, for consistency w/ sibling methods @AndrewLakeMSFT

Reply:
latest Files Changed tab shows h6 now, not h4, for "Parameters" & "Returns" headings for this function, like the others.

rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/dualengine/reference/dualenginesessionfactory?branch=pr-en-us-2859#getspecifiedversion